### DISCUSSION TITLE
Wireless Rear port port

### DISCUSSION THREAD

**Discussion by rasmustjensen** [netbox-community/netbox#10446] [Ideas]
Today only Interfaces can be of type Wireless. Rear port can not be a wireless port.

But I would like to se that you can have a physicals Front Port connected to a Wireless Rear port.

That way you would be able to trace a dedicated path between two devices that are connected with a wireless link.

[Physical port A (_Front port)_ ]**DeviceA**[wireless in/out] (_Rear port_)   - - - - - (_Rear port_)[wireless in/out]**DeviceB**[Physical port B] (_Front port)

So a front port (physical port) which is connected to the Wireless interface.

I see this as the way to model a device "Wireless Outdoor unit" with 2 modules in it. (radio + antenna).
The module "radio" would have the front port and the other module (antenna) would have the rear port (a wireless interface)

Any thoughts?

**Comment by candlerb** [comment]
Frontport/Rearport are for electrically-connected cable pass-through ports only.

Your wireless unit needs two interfaces: a wireless interface, and an ethernet interface.  I believe that you can model the cross-connection by setting the "bridge" attribute on each interface to point to the other one, and that traces should work through this - but I haven't tested it.

**Comment by rasmustjensen** [comment]
Thats a good idea and it works.
Thank you.
![MW link](https://user-images.githubusercontent.com/37774260/191937057-98482267-2d65-426f-bd84-0d89ff20b687.jpg)

**Reply by candlerb** [reply]
Glad it's working.

Out of interest, is your "Wireless Outdoor unit" just an RF amplifier, with an RF coax input?  Or what I mean is, what kind of connection is Cable `#144`?

**Comment by rasmustjensen** [comment]
Yes. Cable "144 and #139  would be Coax RF cable.
Forexample like this one: https://yagionline.com/wp-content/uploads/2021/01/TSR2095120363-1-6990.jpg
Often you would have a short 1m cable connected to the IDU and the and then a thicker and longer (40m) cable interconnected all the way up in the tower where the radio is mounted.
So the system here is a Split mound solution.
(Similar to this Huawei picture:
https://www.daenotes.com/sites/default/files/article-images/split-mount-microwave-equipment.PNG

So in the IDU you have the node processer with modem.
The Modem is then connect the the out door unit which contain both a radio and an antenna.

**Comment by rasmustjensen** [comment]
One downside with this, is that the Bridge part is not visible when looking at the "Connection" 
![Untitled](https://user-images.githubusercontent.com/37774260/193048364-1950ce9a-c278-4bcc-85ab-ae22cd7b9dea.jpg)

**Reply by candlerb** [reply]
Interesting. I would expect the "Link Peers" column to show the immediate neighbour, but it's unclear whether "Connection" should show the very far end as shown by a cable trace (including bridged links), or just the end of the first sequence of cables/frontports/rearports.

Perhaps @jeremystretch could comment as to whether the logic for both is intended to be the same?

For completeness: what version of Netbox are you testing?
